src/main/java/uk/gov/justice/hmpps/casenotes/integrations/PrisonApiService.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.hmpps.casenotes.integrations
+
+import com.fasterxml.jackson.annotation.JsonAlias
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.stereotype.Service
+import org.springframework.web.reactive.function.client.WebClient
+import org.springframework.web.reactive.function.client.bodyToMono
+
+@Service
+class PrisonApiService(@Qualifier("prisonApiWebClient") private val webClient: WebClient) {
+  fun getPrisonSwitch(): List<PrisonDetail> = webClient.get().uri {
+    it.path(GET_PRISON_SWITCH).build(SERVICE_CODE)
+  }.retrieve()
+    .bodyToMono<List<PrisonDetail>>()
+    .retryOnTransientException()
```

Comment:
I've just tried this API call on pre prod and prod where the ALERTS_CASE_NOTES service code has not yet been added and the API returns a 404. If this doesn't already account for that, this code should return an empty list if it gets at 404 response